include/capabilities.h

```diff
@@ -43,6 +43,8 @@ int get_cap_wsize(const struct Capabilities *cap);
 
 int get_cap_xmid(const struct Capabilities *cap);
 int get_cap_ymid(const struct Capabilities *cap);
+int get_cap_xmin(const struct Capabilities *cap);
+int get_cap_ymin(const struct Capabilities *cap);
```

Comment:
It's already here, remove.

Reply:
Hm, I think something messed up when I was doing the rebase. I'll try again later.